pkg/ext-proc/handlers/response.go

```diff
@@ -88,6 +89,11 @@ func (s *Server) HandleResponseBody(reqCtx *RequestContext, req *extProcPb.Proce
 			},
 		},
 	}
+
+	metrics.RecordResponseSizes(reqCtx.Model, reqCtx.ResolvedTargetModel, len(body.ResponseBody.Body))
```

Comment:
I think we should record these in server.go, like other metrics. And server.go also checks if the response is complete (for streaming case). @courageJ

Reply:
Updated